dashboard/lib/projects_list.rb

```diff
@@ -140,7 +140,7 @@ def fetch_section_libraries(section)
             channel_id = storage_encrypt_channel_id(project[:storage_id], project[:id])
             project_owner = section_users.find {|user| user.id == student_storage_ids[project[:storage_id]]}
             project_data = get_library_row_data(project, channel_id, section.name, project_owner)
-            if project_data && (project_owner.user_type == 'student' || project_data[:sharedWith].include?(section.id))
+            if project_data && (project_owner.id != section.user_id || project_data[:sharedWith].include?(section.id))
```

Comment:
this change will still exclude the teacher's libraries (since `section.user_id` is the teacher's ID). i thought we wanted to include the teacher's libraries in this list?

related but separate: lines 129 and 132 (where we assign `student_storage_ids ` and `student_storage_id_list `) should probably be updated since `section_users` includes students and the teacher

Reply:
We want to include all classmates' libraries in the list, but we don't want to include the section owner's libraries in the list.

Details here:
https://docs.google.com/document/d/1zZ6WU5oBfh5Ra5TkdUpYRFsoMpIu9MRCWO42oBMyhrM/edit#

Reply:
ahh, my bad, i had to re-read the slack thread

Reply:
can we cover this case with a unit test to prevent a regression?

Reply:
👍